components/table/hooks/useSorter.tsx

```diff
@@ -130,13 +130,17 @@ function injectSorter<RecordType>(
           className={classNames(`${prefixCls}-column-sorter-up`, {
             active: sorterOrder === ASCEND,
           })}
+          tabIndex={0}
+          role="button"
         />
       );
       const downNode: React.ReactNode = sortDirections.includes(DESCEND) && (
         <CaretDownOutlined
           className={classNames(`${prefixCls}-column-sorter-down`, {
             active: sorterOrder === DESCEND,
           })}
+          tabIndex={0}
+          role="button"
```

Comment:
These arrows is not clickable, you should click on table column header to trigger sorting.

Reply:
It do not makes them clickable. This makes them acceessible (focusable).

Reply:
It will be acceessible and focusable but cannot be handled by keyboard, is it meanless?

Reply:
I agree with @afc163 because when I try out the preview I can navigate to the sort but I can't make it sort with keyboard. So @kpustakhod let me know if you can get it to sort too? If not, our team (including @timgrant-philly and @mellis481) can do the work. Just let us know please?

Reply:
Puched new changes that makes accessible table sorting with keyboard.

https://user-images.githubusercontent.com/16884481/132724103-f0b91d3b-cc76-4dce-a901-42114df07854.mov

Reply:
Looks great. Now how do we get this PR reopened since the github actions not closed it in https://github.com/ant-design/ant-design/pull/32093#issuecomment-916244736